apps/src/p5lab/poetry/locale-do-not-import.js

```diff
@@ -3,6 +3,7 @@
 import safeLoadLocale from '@cdo/apps/util/safeLoadLocale';
 import localeWithI18nStringTracker from '@cdo/apps/util/i18nStringTracker';
 
-let locale = safeLoadLocale('poetry_locale');
-locale = localeWithI18nStringTracker(locale, 'poetry_locale');
-module.exports = locale;
+export const locale = localeWithI18nStringTracker(
```

Comment:
my brain always gets confused by `module.exports` so i think the answer to this is "no," but i want to double check: should this be a default export?

Reply:
I don't think so.